packages/element/src/renderElement.ts

```diff
@@ -659,6 +662,16 @@ const drawElementFromCanvas = (
     // revert afterwards we don't have account for it during drawing
     context.translate(-cx, -cy);
 
+    const isDarkMode = appState.theme === 'dark';
+    let isEmojiImage = false;
+if (element.type === "text" && typeof element.text === "string") {
+  isEmojiImage = isEmojiOnly(element.text);
+}
```

Comment:
Why the complicated prelude? Just embed the test as  const isEmojiOnly and save of LOC